h/static/styles/partials-v2/_search.scss

```diff
@@ -40,19 +40,38 @@
   margin-right: 30px;
 }
 
-.search-result-list {
+.search-result-container--empty {
+  /* The default height of the search container is calculated to so that it
+     stretches to the bottom of the viewport and no further.
+     The px value here is the height of the stuff above the
+     search-result-container. */
+  height: calc(100vh - 124px);
```

Comment:
I'm still seeing a vertical scrollbar on Safari iOS, even though I don't see it on iPhone in Chrome dev tools, but the page seems to be _way_ longer on Safari iOS for some reason, I propose to leave it